<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSetChessboardClass</class>
 <widget class="QDialog" name="DlgSetChessboardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgSetChessboard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>棋盘设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>行：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_row"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>列：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_column"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>特殊块位置设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>x：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_x"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>y：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sBox_y"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pbtn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
